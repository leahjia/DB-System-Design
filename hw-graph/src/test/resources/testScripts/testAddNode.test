# University Map new Node creation testing file.
# Tests that a single node can be created/added to the university map

# Create a graph
CreateGraph map1

# Add a pair of nodes
AddNode map1 n1
AddNode map1 n2
