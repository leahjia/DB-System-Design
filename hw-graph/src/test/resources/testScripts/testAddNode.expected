# University Map new Node creation testing file.
# Tests that a single node can be created/added to the university map

# Create a graph
created graph map1

# Add a pair of nodes
added node n1 to map1
added node n2 to map1
